File: src/main/java/no/nav/fo/forenkletdeploy/config/ApplicationConfig.java
package no.nav.fo.forenkletdeploy.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;
import org.springframework.scheduling.annotation.EnableScheduling;

@Configuration
@Import({
        CacheConfig.class,
        CommonConfig.class,
        ServiceConfig.class
})
@EnableScheduling
public class ApplicationConfig {
}

File: src/main/kotlin/no/nav/fo/forenkletdeploy/config/ApplicationConfig.kt
package no.nav.fo.forenkletdeploy.config

import no.nav.fo.forenkletdeploy.CorsFilter
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.boot.web.servlet.FilterRegistrationBean


@Configuration
open class ApplicationConfig {
    @Bean
    open fun corsFilterRegistration(): FilterRegistrationBean<CorsFilter> {
        val frb: FilterRegistrationBean<CorsFilter> = FilterRegistrationBean()
        frb.filter = CorsFilter()
        frb.urlPatterns = arrayListOf("/*")
        return frb
    }
}
